.github/workflows/regression-test-TT02.yaml

```diff
@@ -0,0 +1,54 @@
+name: Regression test - TT02
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 12 * * 1-5'
+
+jobs:
+  regression-tests:
+    environment: TT02
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
Is this line it's own step without a name?

Reply:
Correct. Usually don't name this simple step because the action itself is pretty clear and we don't have any additional parameters to add either, so would just be increasing the number of lines without adding much value

Reply:
![image](https://github.com/Altinn/altinn-events/assets/47737608/dc82b1cf-3b60-497c-b80b-fe593ea403ec)
 this is what it looks like during a workflow run

Reply:
Okay, thank you.